extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -299,56 +286,90 @@ export class SchemaCompareMainWindow {
-					value: loc.include,
-					cssClass: 'align-with-header',
-					width: 60,
-					type: azdata.ColumnType.checkBox,
-					action: azdata.ActionOnCellCheckboxCheck.customAction
-				},
-				{
-					value: loc.action,
-					cssClass: 'align-with-header',
-					width: 30
-				},
-				{
-					value: loc.targetName,
-					cssClass: 'align-with-header',
-					width: 150
-				}
-			],
-			CSSStyles: { 'left': '15px' },
-			width: '98%'
-		});
-
-		this.splitView.addItem(this.differencesTable);
-		this.splitView.addItem(this.diffEditor);
-		this.splitView.setLayout({
-			orientation: 'vertical',
-			splitViewHeight: 800
-		});
-
 		this.flexModel.removeItem(this.loader);
 		this.flexModel.removeItem(this.waitText);
 		this.resetButtons(ResetButtonState.afterCompareComplete);
 
 		if (this.comparisonResult.differences.length > 0) {
+			const data = this.getAllDifferences(this.comparisonResult.differences);
+
+			this.differencesTable = this.view.modelBuilder.table().withProps({
+				data: data,
+				columns: [
+					{
+						value: loc.type,
+						cssClass: 'align-with-header',
+						width: 50
+					},
+					{
+						value: loc.sourceName,
+						cssClass: 'align-with-header',
+						width: 90
+					},
+					<azdata.CheckboxColumn>
+					{
+						value: loc.include,
+						cssClass: 'align-with-header',
+						width: 60,
+						type: azdata.ColumnType.checkBox,
+						action: azdata.ActionOnCellCheckboxCheck.customAction
+					},
+					{
+						value: loc.action,
+						cssClass: 'align-with-header',
+						width: 30
+					},
+					{
+						value: loc.targetName,
+						cssClass: 'align-with-header',
+						width: 150
+					}
+				],
+				CSSStyles: { 'left': '15px' },
+				width: '98%',
+				title: loc.differencesTableTitle
+			}).component();
+
+			this.diffEditor = this.view.modelBuilder.diffeditor().withProperties({
+				contentLeft: os.EOL,
+				contentRight: os.EOL,
+				height: 500,
+				title: loc.diffEditorTitle
+			}).component();
+
+			let sourceText = '';
+			let targetText = '';
+			this.tablelistenersToDispose.push(this.differencesTable.onRowSelected(() => {
+				let difference = this.comparisonResult.differences[this.differencesTable.selectedRows[0]];
+				if (difference !== undefined) {
+					sourceText = this.getFormattedScript(difference, true);
+					targetText = this.getFormattedScript(difference, false);
+
+					this.diffEditor.updateProperties({
+						contentLeft: sourceText,
+						contentRight: targetText,
+						title: loc.diffEditorTitle
+					});
+				}
+			}));
+			this.tablelistenersToDispose.push(this.differencesTable.onCellAction(async (rowState) => {
+				let checkboxState = <azdata.ICheckboxCellActionEventArgs>rowState;
+				if (checkboxState) {
+					await this.applyIncludeExclude(checkboxState);
+				}
+			}));
+
+			this.splitView = this.view.modelBuilder.splitViewContainer().withItems([
+				this.differencesTable,
+				this.diffEditor
+			]).component();
+
+			// this.splitView.addItem(this.differencesTable);
```

Comment:
do we want to remove the commented ones?

Reply:
Thought I got them all 😄